src/components/Views/SchedulingView.vue

```diff
@@ -1,43 +1,177 @@
+        'max_lunar_phase': 1
+      }
+    }
+  ],
+  'windows': [
+    {
+      'start': formatToUTC(startDate),
+      'end': formatToUTC(endDate)
+    }
+  ],
+  'location': {
+    'telescope_class': '0m4'
+  }
+})
+
+const scheduleObservation = async () => {
+  if (observationData.value) {
+    const requestList = []
+
+    // Handle single target
+    if (observationData.value.target) {
+      const { target, settings, startDate, endDate } = observationData.value
+      requestList.push(createRequest(target, settings, startDate, endDate))
+    }
+
+    // Handle multiple targets
+    if (observationData.value.targets) {
+      const { targets, startDate, endDate } = observationData.value
+      requestList.push(...targets.map(target => createRequest(target, target.exposures, startDate, endDate)))
+    }
+
+    const token = userDataStore.authToken
+    const headers = {
+      'Content-Type': 'application/json',
+      'Accept': 'application/json',
+      'Authorization': `Token ${token}`
+    }
+
+    if (observationData.value.target || observationData.value.targets.length === 1) {
+      operatorValue.value = 'SINGLE'
+    } else if (observationData.value.targets.length > 1) {
+      operatorValue.value = 'MANY'
+    }
+
+    await fetchApiCall({
+      url: 'https://observe.lco.global/api/requestgroups/',
+      method: 'POST',
+      header: headers,
+      body: {
+        'name': 'UserObservation',
+        'proposal': 'LCOSchedulerTest',
+        'ipp_value': 1.05,
+        'operator': operatorValue.value,
+        'observation_type': 'NORMAL',
+        'requests': requestList
+      },
+      successCallback: () => {
+        showScheduled.value = true
+      },
+      failCallback: (error) => {
+        console.error('Error scheduling observation:', error)
+      }
+    })
+  }
+}
+
+const handleUserSelections = (data) => {
+  observationData.value = data
+}
+
+const enableButton = computed(() => {
+  return observationData.value && observationData.value.settings.length > 0
+})
 
 </script>
 
 <template>
-    <div class="container">
-        <div v-if="!level && !showScheduled" class="level-buttons-wrapper">
-            <h2>Schedule an Observation</h2>
-            <p>What level are you?</p>
-            <v-btn @click="level = 'beginner'" color="indigo" class="level-btns">Beginner</v-btn>
-            <v-btn @click="level = 'advanced'" color="indigo" class="level-btns">Advanced</v-btn>
-        </div>
-        <div v-else-if="level === 'beginner' && !showScheduled">
-            <BeginnerScheduling @scheduled="showScheduled = true"/>
-        </div>
-        <div v-else-if="level === 'advanced' && !showScheduled">
-            <AdvancedScheduling @scheduled="showScheduled = true"/>
-        </div>
-        <div v-if="showScheduled">
-            <ScheduledObservations />
-        </div>
+  <div class="container">
+    <div v-if="!level && !showScheduled" class="level-buttons-wrapper">
+      <h2>Schedule an Observation</h2>
```

Comment:
can we say "Submit a Request" instead, since the user is not Scheduling, and they are creating a Request, not an Observation